src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -84,6 +84,11 @@
   -->
   <Import Project="$(MSBuildThisFileDirectory)versioning.targets" Condition="'$(ExcludeVersioningImport)'!='true'" />
 
+  <!--
+    Import the localization target
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)localization.targets" Condition="'$(ExcludeLocalizationImport)'!='true' And Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')" />
```

Comment:
Do you mean put `<ExcludeLocalizationImport Condition="'$(ExcludeLocalizationImport)'=='' And !Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')">true</ExcludeLocalizationImport>` in the PropertyGroup at the top of the file?  I'm not sure if I would consider that more readable, but I'll defer to your opinion on it, as I don't have a lot of experience with MSBuild.